source/includes/read/project.cpp

```diff
@@ -0,0 +1,58 @@
+#include <cstdint>
+#include <iostream>
+#include <vector>
```

Comment:
```suggestion
#include <iostream>
```

Remove unused includes.